lib/compilers/typescript-native.js

```diff
@@ -26,7 +26,7 @@ import {BaseCompiler} from '../base-compiler';
 
 import {TypeScriptNativeParser} from './argument-parsers';
 
-export class TypeScriptCompiler extends BaseCompiler {
+export class TypeScriptNativeCompiler extends BaseCompiler {
     static get key() {
         return 'typescript';
```

Comment:
This could also be changed to typescriptnative, right?

Reply:
Does changing the language key break shortlinks, or do we not store the language in the link data?